utils/config/examples/daos_server_unittests.yml

```diff
@@ -4,6 +4,7 @@ name: daos_server           # sys group daos_server
 access_points: ['example']  # management service leader (bootstrap)
 # port: 10001               # control listen port, default 10001
 provider: ofi+tcp
+nr_hugepages: -1 # disable the use of hugepages when NVMe is not being used
```

Comment:
From a usability perspective, it just doesn't sit right with me that we are expecting the admin to know/understand that they need to set a special configuration value in order to avoid failures in environments where hugepages aren't available. I think we could do a better job of handling this situation in code and not force the admin to deal with it.

In the scope of this PR, my objection to adding this to the examples is that it enshrines the workaround (because really, that's what it is) in documentation as the best practice, and I don't agree that this is a good idea.

Reply:
Thinking about this further: If we decide that we really can't cleanly handle this situation in code, then at the very least we should add a new `disable_hugepages:` config parameter. That is clearly self-documenting and doesn't require the admin to set a strange value that's tied to an implementation detail.

Reply:
thanks for the feedback, I will remove the entry for the moment. could you please create a ticket for the extra config parameter and assign it to me?

- [ ] remove nr_hugepages parameters from example configs